mautic/mautic
# Update ddev dev mode

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ❓ 
| New feature/enhancement? (use the a.x branch)      | ❓ 
| Deprecations?                          | None
| BC breaks? (use the c.x branch)        | ❓  (not as far as I know - depends if folks are using this for prod environments?)
| Automated tests included?              | N/A
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A

#### Description

As far as I know, DDEV is only used as a tool for spinning up development areas in which to test Mautic PRs etc. As such, we should really be defaulting to setting the URL to enable developer mode by default (e.g. index_dev.php after the hostname. Sometimes folks have problems because assets have not been regenerated, for example, because they are testing in prod mode.

@dennisameling maybe you would do a review please?

I am not sure if this would be considered to be a bug fix or a feature, or whether this would be considered a breaking change?

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check to see if the URL opened is dev mode, and all is well.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10986"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
